ONSdigital/eq-author
# Upgrade React Hot Loader

### What is the context of this PR?
Upgrades React Hot Loader to the latest version.
Version 4+ is particularly useful for React 16 features - https://github.com/gaearon/react-hot-loader/releases
### How to review
- Run Tests
- Ensure hot loading still works locally
